### DISCUSSION TITLE
What's different with stumpy and another matrix profile package?

### DISCUSSION THREAD

**Discussion by b-y-f** [TDAmeritrade/stumpy#742] [General]
refering [this one](https://github.com/matrix-profile-foundation/matrixprofile)

**Comment by seanlaw** [comment]
@b-y-f Welcome to the STUMPY community and thank you for your question. STUMPY provides a scalable solution for computing the [matrix profile](https://stumpy.readthedocs.io/en/latest/Tutorial_The_Matrix_Profile.html) for a variety of time series lengths. While the underlying computations are based on [previously published work](https://stumpy.readthedocs.io/en/latest/#references) (by other researchers), there are a few key differences that I think sets STUMPY apart:

1. [Simple and consistent API](https://stumpy.readthedocs.io/en/latest/api.html)
2. [Excellent documentation](https://stumpy.readthedocs.io/en/latest/index.html)
3. Codebase is still being actively maintained
4. An thriving user and developer community
5. Additional supported features (e.g., GPUs, multi-dimensional matrix profiles, and non-normalized matrix profiles, etc)

I hope that answers your question. I highly recommend taking a look at our [in depth tutorials](https://stumpy.readthedocs.io/en/latest/tutorials.html) to learn more.

**Reply by NimaSarajpoor** [reply]
@b-y-f 
Also, based on my personal experience, I should say that it is easier to build stuff with STUMPY once you learn the basics. IMHO, the design is in a way that allows users to easily build new stuff for new application.
